src/crystal/system/unix/random.cr

```diff
@@ -0,0 +1,15 @@
+{% skip_file unless flag?(:unix) && !flag?(:openbsd) && !flag?(:linux) %}
```

Comment:
You don't need to test for Unix flag here, it's already in the unix folder.

Reply:
The whole line is useless because we correctly require in `system/random`, but for some reason (someone doing a stupid `require "./src/*"`) we must prevent files to be parsed, and add duplicates.

So no, we must check for unix.

Reply:
None of the other files in this folder has such a check, there are only `skip_file`s to limit to a specific Unix variant.

If someone tries to require everything in the stdlib's src path, then be it. Live with the errors. I don't see a reasonable cause to safeguard against this.